From 067214b48a01734809774cdf3f3b1823c6dca278 Mon Sep 17 00:00:00 2001
From: Sergio Schvezov <sergio.schvezov@canonical.com>
Date: Mon, 14 Aug 2017 11:18:22 +0000
Subject: [PATCH] Use clang 3.9 by default for Ubuntu 16.04

Sync with clang version selection for coreclr when on Ubuntu 16.04

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>
---
 src/Native/build-native.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/Native/build-native.sh b/src/Native/build-native.sh
index 92ec816..d4caf53 100755
--- a/src/Native/build-native.sh
+++ b/src/Native/build-native.sh
@@ -357,6 +357,11 @@ if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
             __ClangMajorVersion=3
             __ClangMinorVersion=6
         fi
+    elif [[ "$__HostOS" == "Linux" && \
+            "$(lsb_release --id --short)" == "Ubuntu" && \
+            "$(lsb_release --release --short)" == "16.04" ]]; then
+        __ClangMajorVersion=3
+        __ClangMinorVersion=9
     else
         __ClangMajorVersion=3
         __ClangMinorVersion=5
-- 
2.7.4

